Dockerfile

```diff
@@ -97,4 +92,9 @@ RUN yum -y update \
 RUN curl https://packages.microsoft.com/config/rhel/7/prod.repo | tee /etc/yum.repos.d/mssql-release.repo
 RUN ACCEPT_EULA=Y yum install -y msodbcsql17
 
+COPY apollo "${LAMBDA_TASK_ROOT}/apollo"
+ARG code_version="local"
+ARG build_number="0"
+RUN echo $code_version,$build_number > ./apollo/agent/version
+
```

Comment:
Moved this as the last step, so when building locally and making code changes it builds the updated image faster without having to re-install the yum packages